charts/opensearch/README.md

```diff
@@ -52,6 +52,8 @@ helm uninstall my-release
 | `extraVolumes`                     | Array of extra volumes to be added                                                                                                                                                | `[]`                                            |
 | `fullnameOverride`                 | Overrides the `clusterName` and `nodeGroup` when used in the naming of resources. This should only be used when using a single `nodeGroup`, otherwise you will have name conflicts                                                                        | `""`                                            |
 | `hostAliases`                      | Configurable [hostAliases][]                                                                                                                                                                                                                              | `[]`                                            |
+| `hostHttpPort`                         | The http port on the host                                                                                                     | `""`                                          |
+| `hostTransportPort`                         | The transport port on the host                                                                                                     | `""`                                          |
```

Comment:
A more verbose description will help here more.

Reply:
I updated the explanation. Nevertheless an engineer has to know why he/she wants to expose hostPorts and which Envs need to be set for the specific requirement. Maybe you have a better explanation.

Reply:
Can we add some disclaimer so that it is easy for the users to make decision

```
Host IP can change when the container is restarted, two containers using the same hostPort cannot be scheduled on the same node and the usage of the hostPort is considered a privileged operation on OpenShift.
```

Reply:
Where to place the disclaimer? Are you sure to explicit tell people about possible issues in Openshift? Maybe a more general disclaimer would be better. ("Refer to documentation for more information and requirements about using hostPorts.") What are thinking the other contributors about this disclaimer?

Reply:
I am fine with this disclaimer @ph311o , once you add this I am good with this PR?